src/content/configuration/dev-server.md

```diff
@@ -472,6 +473,58 @@ webpack-dev-server --hot-only
 ```
 
 
+## `devServer.http2`
+
+`boolean`
```

Comment:
This means default: `true`

```suggestion
`boolean: true`
```

Reply:
@EugeneHlushko it is not entirely `true` by default, it only defaults to `true` when `https` is enabled, which is not a default. If the `https` option is not included, it will default to `false`

Reply:
In that case we mark it as `false` by default and mention the combination case (which you already did later in the text)
```suggestion
`boolean: false`
```